source/includes/_reserve_page_archive.md

```diff
@@ -0,0 +1,353 @@
+# Reserve Page Archive
+> **Definition**
+
+```
+GET https://demo.ticketswitch.com/f13/reserve_page_archive.v1
+```
+
+This API call should be used in preference to the [status](#status) API call.
```

Comment:
I feel like this sentence will encourage mistakes when integrating. If people don't read further below properly, they will be inclined to use this is the cases where they should have used `status.v1`
Maybe something like 

> This API call should be used in preference to the [status](#status) API call, when you do not care to know about the current status of the reservation.

would be better?

Reply:
Actually, please remove any reference to the status call entirely! Just don't.

What you do need to add is shitloads of warnings about how this produces a verbatim copy of the original page, and thus ignores all HTTP content negotiation stuff passed in as well as any of the usual req_ variables. So if what you originally requested was a page of XML, encoded in a chinese character set, with gzip compression turned on, then that what you will get back form this call, even if this request says it can't support any of those things!

This is, well, not exactly "a hack", but is for a very specific situation where you have called reserve and somehow lost the result. usually because you are handing over to some other piece of code and only want to pass the reference.

So, this also means the example response is nonsense here, as it could be absolutely anything, from HTML to XML to JSON to a 303 redirect. You don't know.

Would suggest you document this and the purchase equivalent together as they do the same thing and the same caveats apply.